internal/cmd/cmd.go

```diff
@@ -119,7 +119,9 @@ func newUseCmd(_ *CLI) *cobra.Command {
 $ infra use development
 
 # Use a Kubernetes namespace context
-$ infra use development.kube-system`,
+$ infra use development.kube-system
+
+# Note: infra:<cluster-name> is equivalent to <cluster-name>`,
```

Comment:
We can remove this help text, since handling `infra:` is more of an exception / user error we're catching here